torch/nestedtensor/nested/creation.py

```diff
@@ -0,0 +1,33 @@
+import torch
```

Comment:
Why not just `torch.nested` (in analogy to `torch.sparse`?)

Reply:
Works for me.